### PR TITLE
allow usage of multiple gopaths

### LINKED ISSUE #123: Doesn't support GOPATH list
Hi! I'm giving a test ride to Kallax and I hit my first minor bump in the road. It's not critical as there is an easy workaround (`export GOPATH=/my/first/go/path`), but it is an inconvenience.

As described [in the Go documentation](https://golang.org/cmd/go/#hdr-GOPATH_environment_variable), GOPATH can be a list (`:`-seperated on UNIX).

But currently the GOPATH is treated as a single value ([here](https://github.com/src-d/go-kallax/blob/master/generator/processor.go#L462) and [here](https://github.com/src-d/go-kallax/blob/master/generator/template.go#L433)).

In my env it generates errors like this : 
```bash
> echo $GOPATH
/my/first/go/path:/my/second/go/path
> go generate .
panic: open /my/first/go/path:/my/second/go/path/src/gopkg.in/src-d/go-kallax.v1/generator/templates/base.tgo: no such file or directory

goroutine 1 [running]:
gopkg.in/src-d/go-kallax.v1/generator.loadTemplateText(0x727b86, 0x12, 0x73a801, 0xc420049ed0)
        /my/first/go/path/src/gopkg.in/src-d/go-kallax.v1/generator/template.go:436 +0xe0
gopkg.in/src-d/go-kallax.v1/generator.makeTemplate(0x711a4e, 0x4, 0x727b86, 0x12, 0x1)
        /my/first/go/path/src/gopkg.in/src-d/go-kallax.v1/generator/template.go:448 +0x39
gopkg.in/src-d/go-kallax.v1/generator.init()
        /my/first/go/path/src/gopkg.in/src-d/go-kallax.v1/generator/template.go:457 +0x126
main.init()
        /my/first/go/path/src/gopkg.in/src-d/go-kallax.v1/generator/cli/kallax/cmd.go:80 +0x58
kallax.go:14: running "kallax": exit status 2
```

### PR DESCRIPTION
**Pull request by erizocosmico**
Fixes #123 

Requires https://github.com/src-d/go-parse-utils/pull/8

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ addons:
 
 env:
   - DBNAME=kallax_test DBUSER=postgres DBPASS=''
+  - GOPATH=$GOPATH:/tmp/whatever
```
Comment (Serabe) [inline .travis.yml]: Invert order.

File: generator/processor.go [hunk 2]
```diff
@@ -459,16 +458,19 @@ func joinDirectory(directory string, files []string) []string {
...
 func typeName(typ types.Type) string {
 	return removeGoPath(typ.String())
 }
 
 func removeGoPath(path string) string {
-	importPath := filepath.ToSlash(goPath + "/src/")
 	path = filepath.ToSlash(path)
-	return strings.Replace(path, importPath, "", -1)
+	for _, p := range parseutil.DefaultGoPath {
+		p = filepath.ToSlash(p + "/src/")
```
Comment (Serabe) [inline generator/processor.go]: Join?

Comment (erizocosmico) [inline generator/processor.go]: Join will strip the last "/", which we want to replace later.

**Comment by erizocosmico** [comment]
Amended with the change, @Serabe

**Pull request merged**